src/github.rs

```diff
@@ -0,0 +1,29 @@
+/*!
+GitHub related logic
+
+# Documentation
+- <https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables>
+- <https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions>
+*/
+
+use std::env;
+
+use once_cell::sync::Lazy;
+
+pub static IS_RUN_AS_GITHUB_ACTION: Lazy<bool> =
```

Comment:
What is the reason for all caps?

Reply:
Normally for being a constant. While `Lazy<Regex>` is run on first usage (sadly can not be done on compile time) I still see it as a constant not changing at runtime.

This might be different as it could be different on every run (whether on GitHub actions or locally)